swupd/manifest.go

```diff
@@ -264,7 +267,7 @@ func ParseManifest(r io.Reader) (*Manifest, error) {
 
 		fields := strings.Split(text, manifestFieldDelim)
 		entry := fields[0]
-		if entry != "includes:" && parsedEntries[entry] > 0 {
+		if !(entry == "includes:" || entry == "optional:") && parsedEntries[entry] > 0 {
```

Comment:
I think the following is easier to read:

if (entry != "includes:" && entry != "optional:") &&........

Reply:
ok, I'll make the change